build.gradle

```diff
@@ -239,6 +239,11 @@ if (project.gradle.startParameter.taskNames.any { it.startsWith("checkPart") ||
 
 subprojects { proj ->
   apply plugin: 'elasticsearch.base'
+
+  repositories {
+    // TODO: Temporary for Lucene RC builds. REMOVE
+    maven { url "https://dist.apache.org/repos/dist/dev/lucene/lucene-9.11.0-RC1-rev-d433394b292e3562e0bb34222f7dd4f307e2b8ca/lucene/maven" }
```

Comment:
Is this still required now that we're pulling in the final version?

Reply:
@mark-vieira this isn't the final version, this is the RC build for the final version. 

I wanted to test all our BWC, etc. against the final build, but alas, now everything is pulling from this url and tests are broken. 

Is there a way for me to update this as an additional optional place to look for binaries so that Elasticsearch can see the release candidate for Lucene?

Reply:
It looks like those failures are because dist.apache.org is unreachable. Some kind of transient network issue? In any case, we can restrict this to say "only use this repo for lucene dependencies":

```groovy
repositories {
  exclusiveContent {
    forRepository {
      maven { url "https://dist.apache.org/repos/dist/dev/lucene/lucene-9.11.0-RC1-rev-d433394b292e3562e0bb34222f7dd4f307e2b8ca/lucene/maven" }
    }
  }
  filter {
    includeGroup "org.apache.lucene"
  }
}
```